api-commons/src/test/java/no/digipost/api/representations/KvitteringsReferanseTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.api.representations;
+
+import org.junit.Test;
+import org.w3.xmldsig.Reference;
+
+import static org.fest.assertions.api.Assertions.assertThat;
+
+public class KvitteringsReferanseTest {
```

Comment:
Hva tenker du om dette nå, @lillesand? Testen for konstruktør med marshaled string blir implisitt avhengig av den som tar inn `Reference`, men vi unngår å ha hardkodede xml-strenger i testen, noe som jeg synes er verre.